### PR TITLE
Update scipy warning

### PR DESCRIPTION
**Pull request by londumas**
Starts fixing https://github.com/igmhub/picca/issues/683, only on warning seen in tests, not others.
Indeed `grep "sp\." -r *` still gives some entries, that either are not triggering warnings or are not tested.

### CODE REVIEW

File: py/picca/utils.py [hunk 1]
```diff
@@ -727,9 +727,9 @@ def unred(wave, ebv, R_V=3.1, LMC2=False, AVGLMC=False):
 
     # Compute optical portion of A(lambda)/E(B-V) curve
     # using cubic spline anchored in UV, optical, and IR
-    xsplopir = sp.concatenate(([0],10000.0/sp.array([26500.0,12200.0,6000.0,5470.0,4670.0,4110.0])))
-    ysplir = sp.array([0.0,0.26469,0.82925])*R_V/3.1
-    ysplop = sp.array((sp.polyval([-4.22809e-01, 1.00270, 2.13572e-04][::-1],R_V ),
+    xsplopir = sp.concatenate(([0],10000.0/np.array([26500.0,12200.0,6000.0,5470.0,4670.0,4110.0])))
```
Comment (andreufont) [inline py/picca/utils.py:730]: should we use numpy concatenate?

Comment (londumas) [reply py/picca/utils.py:730]: This is one of the example that no warning were trigered by the tests. So either it is not tested, or scipy does not give a warning. I would say that it is un safe to change if it is not tested. What do you think ?

Comment (andreufont) [reply py/picca/utils.py:730]: I'd probably leave it for now. May be we can start an Issue about removing Scipy when not necessary, but this is not a priority if they don't cause warnings.

**Comment by londumas** [comment]
@andreufont, thanks for the review. I just merged.

**Pull request merged**